lldb/test/API/functionalities/progress_reporting/swift_progress_reporting/TestSwiftProgressReporting.py

```diff
@@ -8,26 +8,28 @@
 from lldbsuite.test.lldbtest import *
 from lldbsuite.test.decorators import *
 
-class TestSwiftProgressReporting(TestBase):
 
+class TestSwiftProgressReporting(TestBase):
     mydir = TestBase.compute_mydir(__file__)
 
     def setUp(self):
         TestBase.setUp(self)
         self.broadcaster = self.dbg.GetBroadcaster()
-        self.listener = lldbutil.start_listening_from(self.broadcaster,
-                                        lldb.SBDebugger.eBroadcastBitProgress)
+        self.listener = lldbutil.start_listening_from(
+            self.broadcaster, lldb.SBDebugger.eBroadcastBitProgress
+        )
 
     # Don't run ClangImporter tests if Clangimporter is disabled.
-    @skipIf(setting=('symbols.use-swift-clangimporter', 'false'))
+    @skipIf(setting=("symbols.use-swift-clangimporter", "false"))
     @skipUnlessDarwin
     @swiftTest
     def test_swift_progress_report(self):
         """Test that we are able to fetch swift type-system progress events"""
         self.build()
 
-        target, process, thread, _ = lldbutil.run_to_source_breakpoint(self, 'break here',
-                                          lldb.SBFileSpec('main.swift'))
+        target, process, thread, _ = lldbutil.run_to_source_breakpoint(
```

Comment:
Same here.

Reply:
I think this is also a result of `black` formatting.